Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -116,19 +113,6 @@
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toReplaceWith(RawToDigiTask,_hgcal_RawToDigiTask)
 
-# RPC New Readout Validation
-from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
```

Comment:
Do I understand correctly, that with this PR for the 2017 and 2018 runs the muonRPCDigis will only consists in the "unpacker" (legacy config), while in the current CMSSW for 2017 and 2018 the muonRPCDigis is "unpacket+digimerger"?
Is this change of past conditions intended? (Not obvious from your description and slides)

Reply:
Yes, you understand correctly. These periods, 2017 ad 2018, should work with legacy only. When the digimerger was implement  the muonRPCNewDigis and rpcNewRecHits were introduced as a temporally solution, in the sense that the two implementations work in the same time and let the new one (digimerger) be tested.  You pointed out the problem with the MC chain and this PR proposal is to solve it and also to make the new implementation (digimerger) work only for Run3.